config/routes.rb

```diff
@@ -11,6 +11,13 @@
   get '/email/unsubscribe/:uuid' => 'unsubscriptions#confirm', as: :confirm_unsubscribe
   post '/email/unsubscribe/:uuid' => 'unsubscriptions#confirmed', as: :unsubscribe
 
+  if Rails.env != "production" || ENV["GOVUK_APP_DOMAIN"] =~ /integration/
```

Comment:
Do we need to have this restriction in here? Since people will only go to this URL by either typing it in or [if we enable it in Email Alert API](https://github.com/alphagov/email-alert-api/pull/284/files#diff-266065afad82bba6259d1fc97ce5612cR60).

Reply:
I thought we'd decided that we didn't want to risk people finding the page. This way they won't but we can still use it for manual end to end testing on integration. Happy to remove it if you disagree.

Reply:
I don't really mind, I just figured it's an extra level of hiding when we've already got it at the Email Alert API level. It's not enough to stop approval though thinking about it.